packages/clerk-js/webpack.config.js

```diff
@@ -283,13 +326,19 @@ const prodConfig = ({ mode }) => {
     ],
   });
 
-  const clerkCjs = merge(clerkEsm, {
+  const clerkCjs = merge(entryForVariant(variants.clerk), common({ mode }), commonForProd(), commonForProdBundled(), {
```

Comment:
I removed the `merge(clerkEsm` portion because the experimental ESM output was confusing Webpack. It was emitting references to `import.meta.url` within the CJS output, which caused ATTW to complain.